assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -73,19 +75,13 @@ function useUploader() {
    * @param {Object} _additionalData Additional Data object.
    */
   const uploadFile = useCallback(
-    (file, _additionalData = {}) => {
+    async (file, _additionalData = {}) => {
```

Comment:
Seems like uploadFile is used with awaits. Why wasn't this an async function already?

Reply:
Before it didn't contain any asynchronous code

Reply:
https://github.com/google/web-stories-wp/blob/9a31a26e740265b8dc27ca89448df4f0ad3b433c/assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js#L117

Reply:
Before this PR, uploadFile simply returned `uploadMedia(file, additionalData)`, which is a promise. No need to use `async` for that.

In `useUploadMedia` the result of `uploadFile` is used in the middle of a function, so it needs to wait for the Promise to resolve, so that is correct.

All in all not a bug, just JavaScript.